src/lib/checkers.js

```diff
@@ -28,12 +60,24 @@ function findPromotions(configured, retrieved) {
   return promotions;
 }
 
+/**
+ * Find org members who do not have two-factor authentication enabled
+ *
+ * @param {object.<string, object<string, boolean>>} members - User list retrieved from GitHub
+ * @returns {string[]} An array of usernames who are 2FA violators.
+ */
 function validateTwoFactor(members) {
```

Comment:
I assume this is the same object as the `retrieved` above, should it be the same variable name throughout?

Reply:
Possibly! As mentioned above, I think this may warrant some overall rethinking. Which makes me wonder just how much I want to expand the scope of this particular PR...

Reply:
yeah, agreed. Maybe a separate PR that changes things/renames things/whatever your fix is and then continue this PR once that's done?

Reply:
Do you think the order on this is particularly significant? Since I'm doing this solo, I'm somewhat of a mind to get this through first then fix the args. That way I'm not splitting my focus too much.

Reply:
Either order is fine, doing the refactor first might same some cycles on having to redo the jsdoc, but since you're essentially done with the jsdoc I'm not sure it matters either way at this point. completely up to you

Reply:
I was thinking about that. Now that I've got all these well-defined typedefs, I don't think that refactoring the jsdoc would really be all that big of a lift after the fact.